### ISSUE TITLE
Add support for hxfcgi

### ISSUE THREAD

**Issue by njuneau** [njuneau/Harfang#9]
Thanks to ianxm's fork, it may be possible to make Harfang work with hxfcgi. However, due to an incompatibility with macros when using hxfcgi, it may prevent the framework from introducing new features. I will continue to develop the 0.5 branch as usual, and if hxfcgi can be integrated when it is feature-complete, it will be added as an experimental target for Harfang.

**Comment by bablukid** [comment]
why not using this which seems to be more up to date ?
https://medium.com/@sgmacpherson/nginx-tora-haxe-34153741420

**Comment by njuneau** [comment]
That, I think, would be a question better asked to those using hxfcgi ;-) . Unless I'm mistaken, the article you provided explains how to run Tora applications behind Nginx. In this case, the compilation target would be Neko. Since Harfang already supports it, the framework isn't really affected. I may be missing something there, though. Were you having something else in mind? Thanks for the article, by the way!

**Comment by bablukid** [comment]
Maybe we misunderstand each other but I think this issue is pointless since there is already FCGI for the php runtime , and already FCGI for neko (when used with mod_tora instead of mod_neko) ... therefore Harfang or any other haxe-to-neko-or-php web framework can take benefit of these solutions.

**Comment by njuneau** [comment]
No, I believe we are on the same page. The issue may very well be unnecessary now. I would need to check back ianxm's pull request, maybe custom code was required two years ago and it's no longer the case.
